.flutter-plugins

```diff
@@ -0,0 +1,2 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
should this be added to .gitignore? same with flutter-plugins-dependencies according to comments in both those files.